catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -117,18 +118,37 @@ public List<String> listTables(String databaseName) throws NoSuchSchemaException
 
   @Override
   public JdbcTable load(String databaseName, String tableName) throws NoSuchTableException {
+    // We should handle case sensitivity and wild card issue in some catalog tables, take a
+    // MySQL table for example.
+    // 1. MySQL will get table 'a_b' and 'A_B' when we query 'a_b' in a case-insensitive charset
+    // like utf8mb4.
+    // 2. MySQL will view 'a_b' as a wild card, and it will match any table name that starts with
+    // 'a',then any character and then 'b'.
     try (Connection connection = getConnection(databaseName)) {
```

Comment:
Don't you need to escape `databaseName` here?

Reply:
Here is the exact match. Only the method `getTables` is fuzzy matching.
<img width="1488" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/8aa9d30e-64a5-4836-807e-41f9ae68cdea">